google-beta/resource_container_cluster_test.go

```diff
@@ -512,29 +512,6 @@ func TestAccContainerCluster_withPrivateCluster(t *testing.T) {
 	})
 }
 
-func TestAccContainerCluster_withPrivateClusterConfig(t *testing.T) {
```

Comment:
Is this expected?

Reply:
hmm looks like this is a change that @paddycarver added in https://github.com/terraform-providers/terraform-provider-google-beta/commit/52e3f3cfe6c999192d3682d817695e25228a9c2f#diff-aba97f603b034f182c8bbdd928cb3767

I'm guessing it should be here.